tests/daemon/edit.rs

```diff
@@ -26,7 +27,7 @@ async fn create_edited_task(shared: &Shared) -> Result<EditResponseMessage> {
     }
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+#[test(tokio::test(flavor = "multi_thread", worker_threads = 2))]
```

Comment:
Would you recommend to leave this in here?
If so, should we consider adding this to all tests?

Reply:
It shouldn't make much difference if left in place. It'll be helpful for future debugging issues.

I recommend adding it to tests where you feel the need to see log output, on a case by case basis. There is little point in adding it to all the tests just for the sake of it.